.github/workflows/docker.yml

```
@@ -2,56 +2,68 @@ name: Push to DockerHub
 
 on:
   push:
-    tags:
-      - v*
-
-env:
-  REPOSITORY_URL: registry.hub.docker.com
-  SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+    branches: [ 'gr-main' ]
 
 jobs:
-  test:
-    uses: alerta/docker-alerta/.github/workflows/test.yml@master
-    secrets: 
-      DOCKERHUB_TOKEN: ${{ secrets.DOCKERHUB_TOKEN }}
-
   push:
-    needs: test
     runs-on: ubuntu-latest
-    if: github.event_name == 'push'
+    environment: release
     env:
-      IMAGE_NAME: alerta/alerta-web
+      IMAGE_NAME: ${{ github.repository }}
+      WEBUI_SHA: e7575af4f3c38a6cdf9dbd1165b586a0849960c0
     steps:
-      - name: Checkout
+      - name: Checkout docker-alerta
         uses: actions/checkout@v2
-      - name: Variables
-        id: vars
-        run: echo "::set-output name=SHORT_COMMIT_ID::$(git rev-parse --short HEAD)"
-      - name: Build Image
-        id: docker-build
-        run: >-
-          docker build
-          --build-arg VCS_REF=${{ github.sha }}
-          --build-arg BUILD_DATE=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
-          --build-arg VERSION=$(cat VERSION)
-          -t $IMAGE_NAME
-          -t $REPOSITORY_URL/$IMAGE_NAME:$(cat VERSION)
-          -t $REPOSITORY_URL/$IMAGE_NAME:${{ steps.vars.outputs.SHORT_COMMIT_ID }}
-          -t $REPOSITORY_URL/$IMAGE_NAME:latest .
-      - uses: docker/login-action@v1
         with:
-          registry: ${{ env.REPOSITORY_URL }}
-          username: ${{ github.actor }}
-          password: ${{ secrets.DOCKERHUB_TOKEN }}
-      - name: Publish Image
-        id: docker-push
-        run: |
-          docker push $REPOSITORY_URL/$IMAGE_NAME:$(cat VERSION)
-          docker push $REPOSITORY_URL/$IMAGE_NAME:${{ steps.vars.outputs.SHORT_COMMIT_ID }}
-          docker push $REPOSITORY_URL/$IMAGE_NAME:latest
-
-      - uses: act10ns/slack@v1
-        with: 
-          status: ${{ job.status }}
-          steps: ${{ toJson(steps) }}
+          fetch-depth: '0' 
+      - name: Checkout alerta-webui
+        uses: actions/checkout@v2
+        with:
+          repository: 'G-Research/alerta-webui'
+          ref: 'gr-main'
+          fetch-depth: '0'
+          path: 'alerta-webui'
+      - name: Verify SHA
         if: always()
+        working-directory: alerta-webui
+        run: |
+          if [[ "0" -eq $(git merge-base --is-ancestor $WEBUI_SHA HEAD) ]]; then 
+            echo "true"; 
+          else 
+            echo "false"; 
+            exit 1
+          fi
```

This is simpler and is enough to validate the commit SHA is on the `gr-main` branch
```suggestion
        run: git merge-base --is-ancestor $WEBUI_SHA gr-main
```